parametric/protos/apm_test_client.proto

```diff
@@ -76,5 +88,76 @@ message FlushSpansReturn {}
 message FlushTraceStatsArgs {}
 message FlushTraceStatsReturn {}
 
+
+message OtelStartSpanArgs {
+  string name = 1;
+  optional bool new_root = 2;
+  optional uint64 parent_id = 3;
+  optional uint64 span_kind = 9;
+  optional string service = 4;
+  optional string resource = 5;
```

Comment:
Question: how are resource names set in the go implementation? In python otelspan.name == otelspan.resource. Should the otel span object have GetResource/SetResource methods?